conan/tools/scm/git.py

```diff
@@ -31,12 +32,14 @@ def __init__(self, conanfile, folder=".", excluded=None):
             else:
                 self._excluded = conf_excluded
 
-    def run(self, cmd):
+    def run(self, cmd, hidden_output=None):
         """
         Executes ``git <cmd>``
 
         :return: The console output of the command.
         """
+        print_cmd = cmd if hidden_output is None else cmd.replace(hidden_output, "<hidden>")
+        self._conanfile.output.verbose(f"RUN: git {print_cmd}", fg=Color.BRIGHT_BLUE)
```

Comment:
Why not ``info``? Other tools like ``cmake.build()``, using ``self.run()`` will display the CMD in info